src/lib/helpers.ts

```diff
@@ -1,12 +1,26 @@
 import { API_VERSION_HEADER_NAME } from './constants'
-import { SupportedStorage } from './types'
+import { SupportedStorage, User } from './types'
 
 export function expiresAt(expiresIn: number) {
   const timeNow = Math.round(Date.now() / 1000)
   return timeNow + expiresIn
 }
 
 export function uuid() {
+  if (
+    'crypto' in globalThis &&
+    typeof globalThis.crypto === 'object' &&
+    'randomUUID' in globalThis.crypto &&
+    typeof globalThis.crypto.randomUUID === 'function'
+  ) {
+    try {
+      return globalThis.crypto.randomUUID()
+    } catch (e: any) {
+      // ignore and fall back to below implementation, as crypto.randomUUID()
+      // only works in secure contexts
```

Comment:
```suggestion
      // ignore and fall back to below implementation, as crypto.randomUUID()
      // only works in secure contexts (https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts)
```

Reply:
just to confirm, this change isn't related to the getSession stuff right? what is the benefit to try to use `crypto.randomUUID` over our implementation below?

Reply:
Not related, just saw it. It's faster + does not use `Math.random()` which isn't really random.